test/JwtTokenAuthenticatorTest.php

```diff
@@ -229,6 +247,14 @@ public function givenTokenWithoutGroupId_whenVerifyingToken_returnTrue()
     /**
      * @test
      */
+    public function givenTokenWithoutSegments_whenVerifyingToken_returnTrue()
```

Comment:
I don't want to revoke access from users with tokens issued without segments by a previous version of the code. I think right now they would be locked out from preview until they delete their cookies or until their token expires. Most users won't have segments for now anyway.